functions/src/events/createTopicEvent.ts

```diff
@@ -0,0 +1,9 @@
+import { Database } from "../types"
+
+export const createTopicEvent = async (
+  db: Database,
+  eventData: any // Replace 'any' with the appropriate type for your eventData
```

Comment:
Done, I've added the same complex type definition for EventData from components/HearingsScheduled.tsx to functions/src/types.ts, so that it's accessible here.